test/link-check.test.js

```diff
@@ -178,7 +178,7 @@ describe('link-check', function () {
     });
 
     it('should handle file path containing spaces', function(done) {
-        linkCheck('fixtures/s p a c e/A.md', { baseUrl: 'file://' + __dirname }, function(err, result) {
+        linkCheck('A.md', { baseUrl: 'file://' + __dirname + '/fixtures/s p a c e'}, function(err, result) {
```

Comment:
I guess I should have left this test in place (since it tests the file path) and add a new test, that tests the baseUrl.

I can add that to this pull request so that you have coverage for both.